groovy/gradle.java/src/org/netbeans/modules/gradle/java/spi/debug/GradleJavaDebugger.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.gradle.java.spi.debug;
+
+/**
+ * Instance would be found in the project's lookup allow to attach debugger
+ * to the project. This is essentially the same as MavenDebugger for Maven
+ * projects.
+ */
+public interface GradleJavaDebugger {
```

Comment:
Increment module version in `manifest.mf`. Add `@since` tag. Put an entry to `apichanges.xml`.

Reply:
Thank you for that! Just a question on the spec version, after release it bumped from 1.0 to 1.1 on master. So spec version 1.1 is not yet released, do I still need to increase that number?

Reply:
Depends on how precise you want to be? Users of our official releases will be OK with both choices. However if you had users updating NBMs from our daily builder, then they would already have version 1.1 without your API changes - if only the module using the new API was updated, then they wouldn't link properly. I am usually increasing the specification version for every API change.

PS: Dependencies of modules that require the new API should of course be updated as well.